applications/FemToDemApplication/python_scripts/MainFemDem.py

```diff
@@ -0,0 +1,330 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Import system python modules

```

Comment:
This could be moved to analysis

Reply:
in the following PR I'll try to use this "analysis" but maybe it's quite a lot of work

Reply:
![imagen](https://user-images.githubusercontent.com/19991680/31853173-b28ea180-b684-11e7-9a84-d663f7c75d8f.png)